model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -1892,7 +1892,7 @@ class JslDslValidator extends AbstractJslDslValidator {
         if (named.name.length < 2) return;
 
         if (Character.isLowerCase(named.name.charAt(0)) && Character.isUpperCase(named.name.charAt(1))) {
-            error("The first character cannot be lowercase and the second character uppercase.",
+            error("The first character cannot be lowercase and the second character uppercase '"+named.name+"'.",
```

Comment:
Missing spaces

Reply:
uppercase in...

Reply:
https://github.com/BlackBeltTechnology/judo-meta-jsl/pull/140/commits/bfe2c01bd87fdf975f370ad901e05ffd10cb7998